documentation/modules/appendix_crds.adoc

```diff
@@ -277,10 +283,14 @@ It must have the value `oauth` for the type `KafkaListenerAuthenticationOAuth`.
 |xref:type-CertSecretSource-{context}[`CertSecretSource`] array
 |type                            1.2+<.<|Must be `oauth`.
 |string
-|userNameClaim                   1.2+<.<|Name of the claim from the authentication token which will be used as the user principal. Defaults to `sub`.
+|userInfoEndpointUri             1.2+<.<|URI of the User Info Endpoint to use as a fallback to obtaining user id when Introspection Endpoint does not return information that could be used for user id. 
```

Comment:
User Info Endpoint  -- In terms of consistency, we haven't been capitalising endpoint names in the doc. Maybe we could highlight instead: '_user info_ endpoint'. Or express as `userInfoEndpointUri`

Reply:
I don't know. These are standard OAuth 2 endpoints, mentioned in specs. In that sense they are proper names, not common words I guess.